11.0/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN curl -sSL -o java.tar.gz "${URL}" && \
 	sudo ln -s /usr/local/jdk-${JAVA_VERSION}/bin/java /usr/bin/java && \
 	sudo ln -s /usr/local/jdk-${JAVA_VERSION}/bin/javac /usr/bin/javac && \
 	sudo ln -s /usr/local/jdk-${JAVA_VERSION}/bin/javaws /usr/bin/javaws && \
+	sudo ln -s /usr/local/jdk-${JAVA_VERSION} /usr/local/openjdk-11 && \
```

Comment:
Wouldn't this just obliterate the existing directory if it didn't outright fail?

Reply:
That is the same as per the lines above.
e.g. line 15, if /usr/bin/java doesn't exist its created and if it exists already then its obliterated.

I don't understand the fail outright, as the links above don't and it should just be making a link from /usr/local/openjdk-11 to /usr/local/jdk-${JAVA_VERSION}.

I don't care if it is openjdk-11, it can be called bob-11, but I only care it is static per jdk.

It make configuration like maven tool chain far simpler. At the moment for each release, a tool chain xml would need to be updated, but if a symlink was always created then it's one less change.